src/test/java/com/googleinterns/zoomtube/servlets/DiscussionServletTest.java

```diff
@@ -131,11 +178,19 @@ public void doPost_rootComment_storesTypeAndTimestamp() throws ServletException,
     PreparedQuery query = datastore.prepare(new Query(CommentUtil.KIND));
     Comment comment = CommentUtil.createComment(query.asSingleEntity());
     assertThat(comment.parentKey().isPresent()).isFalse();
-    assertThat(comment.timestampMs().isPresent()).isTrue();
-    assertThat(comment.timestampMs().get()).isEqualTo(TIMESTAMP_MS);
+    assertThat(comment.timestampMs().get()).isEqualTo(2000);
     assertThat(comment.type()).isEqualTo(Comment.Type.QUESTION);
   }
 
+  @Test
+  public void doGet_missingLecture_badRequest() throws ServletException, IOException {
+    testServices.setEnvIsLoggedIn(true);
+
+    servlet.doGet(request, response);
+
+    verify(response).sendError(HttpServletResponse.SC_BAD_REQUEST, "Missing lecture parameter.");
```

Comment:
For these raw errors you should add the argument comment /* =*/

Reply:
Added for all 3 PRs